.pa11yci.json

```diff
@@ -1,8 +1,6 @@
 {
-    "defaults": {
-      "runners": [
-        "axe",
-        "htmlcs"
-      ]
-    }
+  "defaults": {
+    "runners": ["axe", "htmlcs"],
+    "hideElements": "li"
```

Comment:
Thanks @rise-erpelding! There is a card to follow up on the linting issue, and I told @bsmrdel101 it was okay to proceed with the `hideElements` config (although we should probably choose a more specific selector than `li`).

For the `package-lock.json`, I think that might be a habit from early in the project when we were adding lots of dependencies. Now that it's more stable, we don't need to run `npm i` so much, and it's probably preferable to run `npm ci` to avoid updating `package-lock.json` unnecessarily.